src/graphics/render-target.js

```diff
@@ -32,6 +32,8 @@ const defaultOptions = {
  * * {@link CUBEFACE_POSZ}
  * * {@link CUBEFACE_NEGZ}
  *
+ * @param {boolean} [options.flipY] - Render the image flipped in Y, useful when the render target will be used as a texture in UI rendering.
```

Comment:
should it mentioned "texture in UI rendering"? It seems like it's useful for using it as a texture in general.

Reply:
Also, specify what it defaults to please.

Reply:
nice one, done. thanks